src/state/info/queries/pools/topPools.ts

```diff
@@ -0,0 +1,59 @@
+import { useState, useEffect } from 'react'
+import { request, gql } from 'graphql-request'
+import { INFO_CLIENT } from 'config/constants/endpoints'
+import { TOKEN_BLACKLIST } from 'config/constants/info'
+import { getDeltaTimestamps } from 'utils/infoQueryHelpers'
+
+interface TopPoolsResponse {
+  pairDayDatas: {
+    id: string
+  }[]
+}
+
+/**
+ * Initial pools to display on the home page
+ */
+const fetchTopPools = async (timestamp24hAgo: number): Promise<string[]> => {
+  try {
+    const query = gql`
+      query topPools($blacklist: [String!], $timestamp24hAgo: Int) {
+        pairDayDatas(
+          first: 5
```

Comment:
topPools and topTokens is temporarily set to 5 cause with 30 it takes forever to load on localhost recently...